package/src/components/Message/Message.tsx

```diff
@@ -698,7 +698,12 @@ const MessageWithContext = <
 
   return (
     <View
-      style={[message.pinned && { backgroundColor: targetedMessageBackground }]}
+      style={[
+        message.pinned && {
+          backgroundColor: targetedMessageBackground,
+          ...targetedMessageContainer,
+        },
+      ]}
```

Comment:
```suggestion
      style={[
        message.pinned && {
          ...targetedMessageContainer,
          backgroundColor: targetedMessageBackground,
        },
      ]}
```

Changing the order here ensures that `targetedMessageBackground` always overrides any background color defined in targetedMessageContainer. I  think that's better as a color could be explicitly defined.,